### PR TITLE
hotfix - repaired ir server which didn't work

### PR DESCRIPTION
**Pull request by RobertLucian**
* Before this `PR`, the `server` which was responsible for interpreting the values off of the `IR sensor` didn't start - there was no procedure for having it start up.
* Made the server start by creating a `service` which boots at startup.
* If the server `fails` during the operation, the `service` tries to restart it in `2 seconds`.

* Now, the user only needs to run `python ~/Dexter/GoPiGo/Software/Python/ir_remote_control/examples/ir_recv_example.py` and **boom**, he has the program up and running in the console.

--------

- [x] Test this `hotfix` on the `SD Card` image that I've been working on.

- [x] Test this `hotfix` on a new fresh `Jessie image` - requires `PR` on `Raspbian for Robots` repo.

- [x] Modify  `README.md` accordingly.

- [x] Team-test this `hotfix` (along with the `README`).

--------

Link to related `PR` on `Raspbian for Robots` : https://github.com/DexterInd/Raspbian_For_Robots/pull/141
Link to related `PR` on `Raspbian For Robots` : https://github.com/DexterInd/Raspbian_For_Robots/issues/149

### CODE REVIEW

**Comment by CleoQc** [comment]
that IR server was not meant to start at boot if I remember correctly but through Advanced Communications Control.
It is incompatible with BrickPi+

**Comment by RobertLucian** [comment]
It didn't start through any means.
The user was supposed to start the server by running `di_ir_reader.py` manually - and nothing was specified in the `README` about this `Python` script. 

The `README` was "expecting" the user to just start the example program and use it.

Also, the install script for the `lirc` (where `lirc` is just a bunch of executables that give access to that `IR Receiver`) was also installing a package called `monit` which is more or less the same thing as `systemd / systemctl combo`. Unfortunately, the `unit file` for `monit` didn't make any reference to the server as it should have done - to me the `unit file` looked as if it was half-complete & half-commented.

File: Software/Python/ir_remote_control/lirc/install.sh [hunk 1]
```diff
@@ -45,49 +45,15 @@ then
 	echo " "
 	echo " "
 
-	echo "Check for internet connectivity..."
```
Comment (DexterInd) [inline Software/Python/ir_remote_control/lirc/install.sh]: Why are we disabling internet connectivity check?

File: Software/Python/ir_remote_control/lirc/install.sh [hunk 2]
```diff
@@ -45,49 +45,15 @@ then
 	echo " "
```
Comment (DexterInd) [inline Software/Python/ir_remote_control/lirc/install.sh]: Where is this install called from when doing a Raspbian for Robots update?  How is it automatically called?

File: Software/Python/ir_remote_control/server/install.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```
Comment (DexterInd) [inline Software/Python/ir_remote_control/server/install.sh]: Where is this called from in Raspbian fo Robots?

Comment (DexterInd) [inline Software/Python/ir_remote_control/server/install.sh]: How does this un-install monit?  If systemd and the monit are both running, will there be collisions?

File: Software/Python/ir_remote_control/lirc/install.sh (hunk 1 above)
Comment (RobertLucian) [inline Software/Python/ir_remote_control/lirc/install.sh]: We don't need any package for installing the `lirc`. Not anymore.

File: Software/Python/ir_remote_control/server/install.sh (hunk 3 above)
Comment (RobertLucian) [inline Software/Python/ir_remote_control/server/install.sh]: The answer is here : https://github.com/DexterInd/Raspbian_For_Robots/pull/141

**Pull request merged**